### ISSUE TITLE
Finish documentation

### ISSUE THREAD

**Issue by GJBoth** [TuragaLab/chromatix#65]

**Comment by schneidermc** [comment]
I've noticed that the Seidel fitting has been added to the examples in the docs. Shall we add the Zernike fitting to the examples, too?

**Comment by diptodip** [comment]
Yes, we'll just need to turn it into a notebook like the Seidel example.

**Comment by schneidermc** [comment]
There should be already an existing notebook. It's possible that we only committed the .py, though. I will check and let you know.

**Comment by schneidermc** [comment]
Regarding the docs, is it possible to add some indents in the table of contents? E.g. have the PhaseMasks heading and then indent the subheadings (PhaseMask, SeidelAberrations, ZernikeAberrations etc.).

**Comment by diptodip** [comment]
> Regarding the docs, is it possible to add some indents in the table of contents? E.g. have the PhaseMasks heading and then indent the subheadings (PhaseMask, SeidelAberrations, ZernikeAberrations etc.).

Yes, this is definitely something I need to fix. We just have to setup mkdocs correctly to handle the headings from the docstrings.

> There should be already an existing notebook. It's possible that we only committed the .py, though. I will check and let you know.

That's right, I think I only got the .py. But I did also update the style of the Seidel notebook a bit. Thanks for checking!

**Comment by diptodip** [comment]
> Regarding the docs, is it possible to add some indents in the table of contents? E.g. have the PhaseMasks heading and then indent the subheadings (PhaseMask, SeidelAberrations, ZernikeAberrations etc.).

So this should be fixed right now (might take some time/refreshing before it changes on readthedocs).
